### DISCUSSION TITLE
The best way to filter employees after they have left

### DISCUSSION THREAD

**Discussion by HNep** [microsoft-search/pnp-modern-search#3985] [General]
I cant use the isactive filter i do have a employee type in entra id which id like to use 1 for is no longer with the company.
Anyone found a way for this?

**Comment by kasperbolarsen** [comment]
For People search I strongly recommend using SharePoint Search as Microsoft people search isn't mature enough. 
This blog post might contain some inspiration for the issue, https://www.m365thinking.com/post/disabled-or-inactive-user-accounts-in-m365-a-prerequisite-for-governance

**Reply by HNep** [reply]
Thanks much the thing is we dont want to really change the firstnames and we dont want all of the users to dissapear when dissabled as we have a large group disabled acoounts which are employed for us. We are also using sharepoint and not Microsoft. Are we able to get the empolyee type to sharepoint from entra through a crawled property?
